tasks/code_sniffer/MailPoet/ruleset.xml

```diff
@@ -88,6 +88,28 @@
       <property name="spacingAfterLast" value="0" /> <!-- no spaces after last method -->
     </properties>
   </rule>
+  <rule ref="Squiz.Functions.FunctionDeclarationArgumentSpacing"> <!-- 1 space between assignment in function declarations -->
+    <properties>
+      <property name="equalsSpacing" value="1" />
+    </properties>
+  </rule>
+  <rule ref="SlevomatCodingStandard.Whitespaces.DuplicateSpaces"> <!-- Checks duplicate spaces anywhere -->
+    <properties>
+      <property name="ignoreSpacesInAnnotation" value="1" /> <!-- to allow multiple spaces to align annotations -->
+      <property name="ignoreSpacesInComment" value="1" /> <!-- to allow multiple spaces to align content of the comment -->
+      <property name="ignoreSpacesBeforeAssignment" value="1" /> <!-- to allow multiple spaces to align assignments -->
```

Comment:
I'm not sure about this rule. If I'm not mistaken it would allow `$x       = 'y';` Correct? I thought that this is something we wanted to disallow. This is used in WP coding style though. Did you discuss it somewhere?

Reply:
I agree with disallowing. I will update the configuration.